### PR TITLE
Amplify Central August Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Create shell for Release Notes

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/central/amplify_relnotes/20210831_ampc_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+The Axway Central CLI extension is compatible with the Axway CLI **version 2.2.0** (<https://www.npmjs.com/package/axway/v/2.2.0>).
+More information about Axway CLI can be found here: [Axway CLI release note](https://docs.axway.com/bundle/axwaycli-open-docs/page/docs/release_notes/2_2_0_20210730_relnotes/index.html).
+
+The following enhancements are available in this Axway Central CLI update:
+
+* When installing an agent, user can select if he wants to send the transactions to the platform or not. He can also decide which level of transactions sampling he wants to be applied and if all errors should be reported.
+* When installing Amplify agents, a new deployment model is available: Helm. This is useful when you need to install the agents on OpenShift or Kubernetes clusters.
+* ISTIO Agents installation follows the same pattern as all other agents: asking questions first, create the Central objects (Service Account / Environment), create additional object into the cluster (Secrets) and last provide the agents starting command.
+
```
Comment (robertengaxway) [inline content/en/docs/central/amplify_relnotes/20210831_ampc_relnotes.md:28]: * Central assets can be viewed as  a filtered list with the use of an RSQL parameter.  For example, use this command to view a list of all assets which have tags equal to tag1 or tag2:   amplify central get assets -q "tags=in=(tag1,tag2)"

File: content/en/docs/central/amplify_relnotes/20210831_ampc_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+### Amplify Central WebUI
+
+The Amplify Central WebUI is used by both the API providers and consumers to manage and consume APIs.
+
+The following enhancements are available in this Amplify Central WebUI update:
+
```
Comment (robertengaxway) [inline content/en/docs/central/amplify_relnotes/20210831_ampc_relnotes.md:34]: Service Mesh support has been updated to use environments with connected Istio Agents in the Topology view.

**Review by cbordier — APPROVED** [review]
I approve the changes.

**Review by robertengaxway — APPROVED** [review]
I approve of the release notes

**Comment by cbordier** [comment]
@lbadenhop, robert and I approved the changes. You can merge it.

**Comment by lbadenhop** [comment]
@Cyrille Please review my edits before I merge

**Comment by cbordier** [comment]
@lbadenhop, I think "to" or "into" is missing in this sentence:
 The customer can upload the report **into** Amplify platform.

**Pull request merged**